frontend/app/model/Faq.scala

```diff
@@ -146,4 +146,47 @@ object Faq {
     )
   )
 
+  val subscribers = List(
+    Item("Can I cancel membership during my trial?",
+      Html("Yes, There is no obligation to continue to a paid Membership after your free 6 months and you can cancel at any time during the trial period."),
+      "cancel-trial"
+    ),
+    Item("I'm a subscriber and I'm also a member, will I get this offer too?",
+      Html(s"Yes, you may be entitled to receive this offer, please call customer services team on 0330 333 6898 or email <a href='mailto:${Config.membershipSupport}'>${Config.membershipSupport}</a>"),
+      "existing-subscriber-member"
+    ),
+    Item("How much does membership cost normally?",
+      Html("The Friend tier of Guardian Membership is free. Partners pay £15 per month or £135 per year if you pay in one go. For Patrons, the cost is £60 per month or £540 per year. If you choose the annual payment you get 3 months free when compared to the monthly price."),
```

Comment:
Don't these pricing values live in a config file somewhere? If so, might be nice to use them here to save cleaning them up if we change prices.